.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,28 @@
+<!-- 
+Insert issue number that this PR fixes (if any) just after 'Fixes #'.
+If this PR does not fix an existing issue, consider opening one or remove 'Fixes #' from the PR description.
+-->
 Fixes #
 
 Changes made in this Pull Request:
- - 
+<!-- Describe the changes that this PR makes. If applicable, use the following bullet list. --> 
+-
 
-
-PR Checklist
-------------
- - [ ] Tests?
- - [ ] Docs?
- - [ ] CHANGELOG updated?
+## PR Checklist
+<!-- Please use the following checklist to ensure the PR is ready to be reviewed/merged. -->
  - [ ] Issue raised/referenced?
+ - [ ] Tests updated/added?
+ - [ ] Documentation updated/added?
+ - [ ] `package/CHANGELOG` file updated?
+ - [ ] Is your name in `package/AUTHORS`? (If it is not, add it!)
+
+## Developers Certificate of Origin
+<!-- 
+CHECKING THE FOLLOWING BOX (BY ADDING 'x' INSIDE THE BRACKETS `[x]`) IS MANDATORY FOR ALL PRs.
+-->
+
+In order for this PR to be merged you **must certify that you are able to submit your code to be included in MDAnalysis**. Read the links and then check the box if you agree.
+
+Code in MDAnalysis is under [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE).
```

Comment:
```suggestion
Code in MDAnalysis is licensed under [LGPLv2+](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE).
```
Or something like that? This isn't a "content" comment, but rather a "that sentence flows weird when I read it", so it could just be me.

Reply:
I think the idea here was to not mention the specific license directly. But maybe it's better to do so. Otherwise I can try to find an alternative phrasing which allows to link to the license file without explicitly mentioning the license.